### ISSUE TITLE
Keymapping on Ubuntu

### ISSUE THREAD

**Issue by C-Bouthoorn** [noseglid/atom-build#378]
I've edited the keymap manually for a few updates now, but I think it's better to implement this globally.
On Ubuntu, `CTRL-ALT-T` opens a terminal, so when I try to select a target, it fails. I've manually changed this to `CTRL-ALT-N` to keep the `BVGHN` keys for build. (They're close to each other on a QWERTY keyboard). Maybe we could change this for everyone, as I suspect a few people will be using Ubuntu (or another distro that has `CTRL-ALT-T` assigned to opening a terminal on default, for example Debian)

**Comment by noseglid** [comment]
You wouldn't have to change it for every update. Rather change your keymaps (Command palette -> `Application: Open Your Keymap`) and add

```
".platform-linux atom-workspace, .platform-linux atom-text-editor":
    '"ctrl-alt-n": "build:select-active-target"
```

And it will persist updates. Of course you can add more custom keybindings. See [keymaps/build.json](https://github.com/noseglid/atom-build/blob/master/keymaps/build.json) for the defined ones.

**Comment by jhasse** [comment]
This has already been fixed in #215. Try `F7` to select the target :)

**Comment by C-Bouthoorn** [comment]
Oh wow @jhasse someone obviously didn't RTFM. Thanks! Closing this
